src/test/java/picard/util/GCloudTestUtils.java

```diff
@@ -10,9 +10,12 @@ public final class GCloudTestUtils {
      */
     public static final PicardHtsPath REQUESTER_PAYS_BUCKET_DEFAULT = new PicardHtsPath("gs://hellbender-requester-pays-test/");
 
-    public static final PicardHtsPath TEST_INPUTS_DEFAULT = new PicardHtsPath("gs://hellbender/test/resources/");
-    public static final PicardHtsPath TEST_STAGING_DEFAULT = new PicardHtsPath("gs://hellbender-test-logs/staging/");
-    public static final PicardHtsPath TEST_PROJECT_DEFAULT = new PicardHtsPath("broad-dsde-dev/");
+    public static final String TEST_INPUTS_DEFAULT_STR = "gs://hellbender/test/resources/";
+    public static final String TEST_STAGING_DEFAULT_STR = "gs://hellbender-test-logs/staging/";
+    public static final String TEST_PROJECT_DEFAULT_STR = "broad-dsde-dev";
+    public static final String TEST_OUTPUT_DEFAULT_STR = TEST_STAGING_DEFAULT_STR + "picard/";
+    public static final PicardHtsPath TEST_INPUTS_DEFAULT = new PicardHtsPath(TEST_INPUTS_DEFAULT_STR);
```

Comment:
I know some of these were just moved, but in some places, you used a `_CLOUD` or `_GCLOUD` in constant names like these, which nicely reinforces that they're remote - I'd suggest doing that for these as.

Reply:
done